<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>331</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Tab1">
      <attribute name="title">
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBoxNetwork">
         <property name="title">
          <string> Network </string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout3">
            <item row="0" column="0">
             <widget class="QLabel" name="labelIP">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Address:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditIP"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelPort">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Port:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBoxPort">
              <property name="maximum">
               <number>65000</number>
              </property>
              <property name="value">
               <number>5660</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QCheckBox" name="checkBoxActivityTimer">
              <property name="text">
               <string>Activity timer</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout4">
         <item>
          <widget class="QCheckBox" name="checkBoxRotatorWindowFrame">
           <property name="text">
            <string>Rotator window frame visible (reboot)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxConnectOnStartup">
           <property name="text">
            <string>Connect on startup</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxRotatorWindowStartOnStop">
           <property name="text">
            <string>Rotator window start on top</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxRotatorShowMousePointer">
           <property name="text">
            <string>Show mouse pointer (Rotator window)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Tab2">
      <attribute name="title">
       <string>PowerMeter</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBoxPowerMeterNetwork">
         <property name="title">
          <string>Enabled</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayoutPowerMeter3">
            <item row="0" column="0">
             <widget class="QLabel" name="labelPowerMeterIP">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Address:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelPowerMeterPort">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Port:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEditPowerMeterIP"/>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBoxPowerMeterPort">
              <property name="maximum">
               <number>65000</number>
              </property>
              <property name="value">
               <number>5660</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxPowerMeterBand">
           <item>
            <property name="text">
             <string>160m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>80m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>40m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>30m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>20m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>17m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12m</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10m</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelPowerMeterScaling">
           <property name="text">
            <string>Scaling</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPowerMeterAddress">
           <property name="text">
            <string>Address:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxPowerMeterAddress">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QLabel" name="labelPowerMeterAddressDec">
           <property name="text">
            <string>(dec, 0 = disabled)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QCheckBox" name="checkBoxPowerMeterAllBands">
           <property name="text">
            <string>All bands</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="comboBoxPowerMeterScaling">
           <item>
            <property name="text">
             <string>Auto</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>150W</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1000W</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1500W</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2000W</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3000W</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5000W</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>10000W</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>15000W</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelPowerMeterBand">
           <property name="text">
            <string>Band:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_14">
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxPowerMeterUpdateRateText">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelPowerMeterUpdateRateBargraphDec">
           <property name="text">
            <string>(ms)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxPowerMeterUpdateRateBargraph">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>5000</number>
           </property>
           <property name="singleStep">
            <number>100</number>
           </property>
           <property name="value">
            <number>200</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="labelPowerMeterUpdateRateTextDec">
           <property name="text">
            <string>(ms)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelPowerMeterUpdateRateText">
           <property name="text">
            <string>Text:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelPowerMeterUpdateRateBargraph">
           <property name="text">
            <string>Bargraph:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout4">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>groupBoxNetworkClicked(bool)</slot>
  <slot>groupBoxCOMClicked(bool)</slot>
 </slots>
</ui>
